### PR TITLE
i have done api user profil fot github stats

### PR DESCRIPTION
**Pull request by DavidLoredan**

### CODE REVIEW

**Review by Hacene010 — DISMISSED** [review]

File: src/components/UserInfo.jsx [hunk 1]
```diff
@@ -1,28 +1,27 @@
 import '../reset.css';
 import './UserInfo.scss';
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
 
 function UserInfo() {
+  const [users, setUsers] = useState([]);
```
Comment (LoicBrassart) [inline src/components/UserInfo.jsx]: You fetch only *one* user -> `[user, setUser]`

File: src/components/UserInfo.jsx [hunk 2]
```diff
@@ -1,28 +1,27 @@
...
-          </li>
-          <li>
-            <span className="numberOfRepository">413</span> repository
-          </li>
-          <li>john.do@contact.io</li>
-        </ul>
-      </section>
+    <div className="UserInfo">
+      <img className="userAvatar" srcSet={users.avatar_url} alt="Avatar" />
+      <></>
```
Comment (LoicBrassart) [inline src/components/UserInfo.jsx]: This has absolutely no effect, delete it

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Minor Evils

File: src/components/UserInfo.jsx (hunk 1 above)
Comment (DavidLoredan) [reply src/components/UserInfo.jsx]: okay, no problem i will replace this immediately

**Review by Hacene010 — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
When I ask for fixes, please don't add more code: just do the fixes ;-)

**Pull request merged**